Control/Monad/Identity.hs

```diff
@@ -35,13 +35,7 @@ version of that monad.
 module Control.Monad.Identity (
     module Data.Functor.Identity,
     module Control.Monad.Trans.Identity,
-
-    module Control.Monad,
-    module Control.Monad.Fix,
    ) where
 
-import Data.Functor.Identity
+import Data.Functor.Identity (Identity, runIdentity)
```

Comment:
After this change, `Control.Monad.Identity` no longer re-exports the `Identity` data constructor (only the type constructor). Was this intended? (I ran into this issue in practice when trying to test some packages for `mtl-2.3` compatibility.)

Reply:
No. Let me fix this

Reply:
That said, I'd recommend to import `Identity` from `Data.Functor.Identity`.